renderer/actions/EditorActions.ts

```diff
@@ -47,3 +47,13 @@ export const downloadCode: EditorActions['downloadCode'] = () => ({
 export const uploadCode: EditorActions['uploadCode'] = () => ({
   type: consts.EditorActionsTypes.UPLOAD_CODE,
 });
+
+export const updateBitmap: EditorActions['updateBitmap'] = (bitmap: number) => ({
+  type: consts.EditorActionsTypes.UPDATE_BITMAP,
+  bitmap,
+})
```

Comment:
Let's rename this action to `updateKeyboardBitmap` just to be clearer on what the bitmap is for.